src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -115,6 +115,32 @@ def load_arguments(self, _):
         c.argument('conflict_resolution_policy', options_list=['--conflict-resolution-policy', '-c'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Conflict Resolution Policy, you can enter it as a string or as a file, e.g., --conflict-resolution-policy @policy-file.json or ' + SQL_GREMLIN_CONFLICT_RESOLUTION_POLICY_EXAMPLE)
         c.argument('throughput', help='The throughput of SQL container (RU/s). Default value is 400')
 
+# SQL stored procedure
+    with self.argument_context('cosmosdb sql stored_procedure') as c:
+        c.argument('account_name', options_list=['--account-name', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database-name', '-d'], help="Database name")
+        c.argument('container_name', options_list=['--container-name', '-c'], help="Container name")
+        c.argument('stored_procedure_name', options_list=['--name', '-n'], help="StoredProcedure name")
+        c.argument('stored_procedure_body', options_list=['--body', '-b'], help="StoredProcedure body")
```

Comment:
Can you make the help message more descriptive?vFor example, which kind of value is valid for storedprocedure body? The body value is string with any value or it should be in json format?